dvc/command/checkout.py

```diff
@@ -26,12 +26,13 @@ def _human_join(words):
 
 def log_summary(stats):
     states = ("added", "deleted", "modified")
-    summary = (
+    summary = [
         "{} {}".format(len(stats[state]), state)
         for state in states
         if stats.get(state)
-    )
-    logger.info(_human_join(summary))
+    ]
+    if summary:
+        logger.info(_human_join(summary))
```

Comment:
Maybe it is worth putting some message that will say that there was nothing to do?

Reply:
Makes sense. Thanks.